java/emitter.js

```diff
@@ -1,425 +1,484 @@
+                }
+            }
+        },
+        notEqual(actual, expected, message) {
+            addCommand(java`Assert.assertNotEquals(${actual}, ${expected}${assertMessage(message)});`)
+        },
+        instanceOf(object, typeName) {
+            addCommand(java`Assert.assertTrue(${object} instanceof ${insert(types[typeName].name())});`)
+        },
+        throws(func, check) {
+            let command
+            if (check) {
+                command = java`Assert.assertThrows(${insert(check())} , new Assert.ThrowingRunnable(){
           public void run() {${func}}
         });`
-      } else {
-        command = java`Assert.assertThrows(new Assert.ThrowingRunnable(){ 
+            } else {
+                command = java`Assert.assertThrows(new Assert.ThrowingRunnable(){ 
         public void run() {${func}}
         });`
-      }
-      addCommand(command)
-    },
-    ok(arg, message) {
-      addCommand(java`Assert.assertNotNull(${arg}${extraParameter(message)});`)
+            }
+            addCommand(command)
+        },
+        ok(arg, message) {
+            addCommand(java`Assert.assertNotNull(${arg}${assertMessage(message)});`)
+        }
     }
-  }
 
-  const helpers = {
-    getTestInfo(result) {
-      return addCommand(java`getTestInfo(${result});`).type({
-        type: 'TestInfo',
-        schema: {
-          actualAppOutput: {
-            type: 'Array',
-            items: {
-              type: 'AppOutput',
-              schema: {
-                image: {
-                  type: 'Image',
-                  schema: {hasDom: 'Boolean', location: "Location"},
+    const helpers = {
+        getTestInfo(result) {
+            return addCommand(java`getTestInfo(${result});`).type({
+                type: 'TestInfo',
+                schema: {
+                    actualAppOutput: {
+                        type: 'Array',
+                        items: {
+                            type: 'AppOutput',
+                            schema: {
+                                image: {
+                                    type: 'Image',
+                                    schema: {
+                                        hasDom: 'Boolean',
+                                        location: Location
+                                    },
+                                },
+                                imageMatchSettings: ImageMatchSettings,
+                                knownVariantId: {
+                                    type: 'String'
+                                },
+                                pageCoverageInfo: {
+                                    type: 'PageCoverageInfo',
+                                    schema: {
+                                        pageId: 'String',
+                                        width: 'Long',
+                                        height: 'Long',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8528488)
```suggestion
                                        height: "Long",
```